src/main/java/com/blackducksoftware/integration/hub/notification/content/detail/NotificationContentDetailFactory.java

```diff
@@ -52,32 +53,33 @@ public NotificationContentDetailFactory(final Gson gson, final JsonParser jsonPa
     }
 
     public List<NotificationContentDetail> generateUserContentDetails(final NotificationUserView view) {
-        return generateContentDetails(view.contentType, view.createdAt, view.notificationState, view.type, view.json);
+        return generateContentDetails(view.contentType, view.createdAt, Optional.of(view.notificationState), view.type, view.json);
```

Comment:
Is view.notificationState guaranteed to not be null? If it can be null, we should use Optional.ofNullable()

Reply:
In this case, we are dealing with a user notification which will always have a notification state.